tests/test_dryer.py

```diff
@@ -0,0 +1,88 @@
+from collections.abc import Callable
+from typing import Any
+
+import pytest
+from aioresponses import aioresponses
+from yarl import URL
+
+from whirlpool.appliancesmanager import AppliancesManager
+from whirlpool.auth import Auth
+from whirlpool.backendselector import BackendSelector
+from whirlpool.dryer import MachineState
+
+
+async def test_attributes(appliances_manager: AppliancesManager):
+    dryer1 = appliances_manager.dryers[0]
+    assert dryer1.get_machine_state() == MachineState.Standby
+    assert dryer1.get_door_open() is False
+    assert dryer1.get_est_time_remaining() == 1800
+    assert dryer1.get_drum_light_on() == 0
+    assert dryer1.get_status_extra_steam_changeable() == 1
+    assert dryer1.get_status_cycle_select() == 0
+    assert dryer1.get_status_dryness() == 1
+    assert dryer1.get_status_manual_dry_time() == 1
+    assert dryer1.get_status_temperature() == 1
+    assert dryer1.get_status_wrinkle_shield() == 1
+    assert dryer1.get_dryness() == 10
+    assert dryer1.get_manual_dry_time() == 1800
+    assert dryer1.get_cycle_select() == 11
+    assert dryer1.get_airflow_status() == 0
+    assert dryer1.get_cool_down() == 0
+    assert dryer1.get_damp() == 0
+    assert dryer1.get_drying() == 0
+    assert dryer1.get_limited_cycle() == 0
+    assert dryer1.get_sensing() == 0
+    assert dryer1.get_static_reduce() == 0
+    assert dryer1.get_steaming() == 0
+    assert dryer1.get_wet() == 0
+    assert dryer1.get_cycle_count() == 195
+    assert dryer1.get_running_hours() == 148
+    assert dryer1.get_total_hours() == 6302
+    assert dryer1.get_rssi_antenna_diversity() == -51
+    assert dryer1.get_damp_notification_tone_volume() == 0
+    assert dryer1.get_alert_tone_volume() == 0
+    assert dryer1.get_temperature() == 2
+    assert dryer1.get_wrinkle_shield() == 0
+
+
+@pytest.mark.parametrize(
+    ["method", "argument", "expected_json"],
+    [
+    ],
```

Comment:
the params are missing, so the test is not running

Reply:
I'll add some properties to set.  But, I'll have to guess for the washer.

Reply:
I haven't added any settable parameter to the dryer class and the washer class doesn't have any either.